x/mock/simulation/constants.go

```diff
@@ -28,4 +28,10 @@ var (
 		{10, 50, 5},
 		{0, 10, 1000},
 	})
+	// 3 states: rand in range [0, 4*provided blocksize], rand in range [0, 2 * provided blocksize], 0
+	blockSizeTransitionMatrix, _ = CreateTransitionMatrix([][]int{
```

Comment:
Are there docs for these matrices? It's not immediately clear to me what this is and how I should use it as a contributing developer.

Reply:
I think the transition matrix is well documented in it's own file. Should I just put a comment referencing that these are transition matrices?

Reply:
no, thats okay. I guess I was just being lazy in that I did not look at that documentation.